roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -114,12 +110,21 @@
     command: swapoff --all
 
   - name: Remove swap entries from /etc/fstab
+    replace:
+      dest: /etc/fstab
+      regexp: '(^[^#].*swap.*)'
+      replace: '# \1'
+      backup: yes
+
+  - name: Add notice about disabling swap
     lineinfile:
       dest: /etc/fstab
-      regexp: 'swap'
-      state: absent
+      line: '# OpenShift-Ansible Installer disabled swap per overcommit guidelines'
+      state: present
 
-  when: swap_result.stdout_lines | length > 0
+  when:
+  - swap_result.stdout_lines | length > 0
+  - openshift_disable_swap | default(true)
```

Comment:
I think I've encountered travis complaining when the when conditions weren't indented, any idea why it seems to be inconsistent? Here they're not intended, but in openshift_node/tasks/main.yml they are.